float32 linearVelocityX
float32 angularVelocityZ
